README.md

```diff
@@ -33,6 +33,7 @@ Initialize your Wasm Build environment (*required for compiling Rust to Wasm*):
 
 Build Wasm and native code:  
 ```bash
+cd node
```

Comment:
Why do you need to go to the subfolder? it works fine from root no ?

Reply:
Not necessary, just removed it.